framework/SupervisedLearning/ARMA.py

```diff
@@ -596,6 +595,36 @@ def _generateVARMASignal(self, model, numSamples=None, randEngine=None, rvsIndex
                                      state_shocks = stateShocks)
     return obs
 
+  def getRomClusterValues(self, featureTemplate, *args, **kwargs):
+    """
+      Indicates the parameters on which this ROM can cluster.
+      @ In, featureTemplate, str, format for feature space names (takes target, separator, metric, and id)
+      @ Out, features, dict, cluster feature values
+    """
+    # algorithm for providing Fourier series and ARMA white noise variance and #TODO covariance
+    features = {}
+    # include Fourier if available
+    for target in self.fourierResults:
+      for b,base in enumerate(self.fourierResults[target]['regression']['coeffs']):
+        for s,subdivision in enumerate(base):
+          for waveform, coeff in subdivision.items():
+            ID = '{}_{}_{}'.format(b,s+1,waveform)
+            feature = featureTemplate.format(target=target, metric='Fourier', id=ID)
+            features[feature] = coeff
+    # signal variance, ARMA (not varma)
+    for target,arma in self.armaResult.items():
+      feature = featureTemplate.format(target=target, metric='arma', id='std')
+      features[feature] = np.sqrt(arma.sigma2)
+    return features
+
+  def getRomClusterParams(self):
+    """
+      Indicates the parameters on which this ROM can cluster.
+      @ In, None
+      @ Out, params, list, clusterable parameters
```

Comment:
docstring

Reply:
I don't know what you didn't like. Hopefully I satisfied the request.